app/src/main/java/com/android/doctorapp/ui/doctor/UpdateDoctorProfileFragment.kt

```diff
@@ -96,6 +96,10 @@ class UpdateDoctorProfileFragment :
     private var tempShiftTimeList = ArrayList<AddShiftTimeModel>()
 
     lateinit var bottomSheetFragment: BottomSheetDialog
+
+    lateinit var updateProfileCalender: Calendar
```

Comment:
Define calendar object in baseFragment show any fragment can easily access and no need to create each every fragment inside it instance.

Like     var myCalender: Calendar = Calendar.getInstance()

Reply:
Done